perun-core/src/main/resources/postgresChangelog.txt

```diff
@@ -6,6 +6,11 @@
 -- Directly under version number should be version commands. They will be executed in the order they are written here.
 -- Comments are prefixed with -- and can be written only between version blocks, that means not in the lines with commands. They have to be at the start of the line.
 
+3.1.98
+create table created_app_notifications_sent (app_id int not null, created_at timestamp default statement_timestamp() not null, created_by varchar default user not null, constraint appnotifsent_pk primary key(app_id), constraint appnotifsent_app_fk foreign key (app_id) references application(id) on delete cascade);
```

Comment:
What about already existing applications? How they will be treated when we deploy this feature?

Reply:
Submitted group applications would be considered that no email was sent after their submission, but it might have been sent if the member already existed and was in a VALID or INVALID state (it was an EXTENSION group application).
Then we deploy these changes, and the email will be always sent when the VO application is approved. In the case of such EXTENSION applications, I believe the email would be indeed sent twice, once before deployment and once after.